modules/tensor_mechanics/include/utils/EulerAngles.h

```diff
@@ -9,6 +9,7 @@
 #define EULERANGLES_H
 
 #include "MooseTypes.h"
+#include "MooseRandom.h"
```

Comment:
forward declare here `class MooseRandom;`

Only include in the header what you _absolutely must_ include. This impacts compile time.